src/map/mob.c

```diff
@@ -3733,74 +4010,126 @@ bool mob_parse_dbrow(char** str) {
 #endif
 
 	//Tests showed that chase range is effectively 2 cells larger than expected [Playtester]
-	if (db->range3 > 0)
-		db->range3 += 2;
+	if (entry->range3 > 0)
```

Comment:
Considering how this code has never been in use (it was misplaced since the beginning), removing it is the less impacting option I guess, especially considering how much of an undocumented hack it is. Oh, why didn't I start earlier to reject those large and messy, I wonder.
Whether it really is needed or not, only @Michieru can tell, since he's the original committer.